rollup.config.mjs

```diff
@@ -1,7 +1,9 @@
-import pkg from './package.json' assert { type: 'json' };
```

Comment:
this was broken with my version of node, so i replace with:

```mjs
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const pkg = require('./package.json');
```

which seems safier